R/openapi-spec.R

```diff
@@ -40,15 +40,44 @@ endpointSpecification <- function(routerEndpointEntry, path = routerEndpointEntr
 }
 
 defaultResponse <- list(
+  "200" = list(
+    description = "OK"
+  ),
+  "500" = list(
+    description = "Internal Server Error",
+    content = list("text/plain" = list(schema = list(type = "string")))
```

Comment:
Updated in recent PR to serialize using `serializer_unboxed_json()`

```suggestion
    content = list("application/json" = list(schema = list(type = "string")))
```